DEVDOC.md

```diff
@@ -152,14 +152,17 @@ To use the delve debugger for Go with VSCode to debug `gapis`. These steps can b
     ...
 }
 ```
-A sample `<host>` can be `127.0.0.1` and `<port>` can be `1234`.
+An example `<host>` can be `127.0.0.1` and `<port>` can be `1234`.
 
 4. Start delve in headless mode at gapid check-in folder.
 ```
 dlv exec --headless --listen=<host>:<port> --api-version 2 ./bazel-bin/pkg/gapis -- <gapis-arguments>
 ```
 
-This will allow using port `1234` (or any other preferred port) to connect to delve from VSCode.
+Below commnad will allow using port `1234` (or any other preferred port) to connect to delve from VSCode.
```

Comment:
typo "commnad", and maybe rephrase: "The command below will allow using ..."

Reply:
+1